### PR TITLE
Add Misol HP2250 Pro to the device map

### PR DESCRIPTION
**Pull request by xrh0905**
**Describe what the PR does:**

Add a new device

**Does this fix a specific issue?**

Fixes https://github.com/bachya/ecowitt2mqtt/issues/206

**Checklist:**

- [ ] Confirm that one or more new tests are written for the new functionality.
- [x] Run tests and ensure everything passes (with 100% test coverage).
- [ ] Update `README.md` with any new documentation.
- [ ] Add yourself to `AUTHORS.md`.

### CODE REVIEW

File: ecowitt2mqtt/helpers/device.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "GW2000A": ("Ecowitt", "GW2000A"),
     "GW2000B": ("Ecowitt", "GW2000B"),
     "PT-HP2550": ("Fine Offset", "HP2550"),
+    "HP2550_Pro": ("Misol", "HP2250_Pro"),
```
Comment (bachya) [inline ecowitt2mqtt/helpers/device.py:20]: Small not: please keep the keys in alphabetical order.

**Review by bachya — CHANGES_REQUESTED** [review]
Thanks! Small comment, then we can merge.

File: ecowitt2mqtt/helpers/device.py (hunk 1 above)
Comment (xrh0905) [reply ecowitt2mqtt/helpers/device.py:19]: Done

**Review by bachya — APPROVED** [review]
Thank you!

**Pull request merged**